[2023-09-04T06:07:17.601+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2023-09-04T06:07:17.631+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2023-09-04T06:07:17.631+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:07:17.697+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-08-29 00:00:00+00:00
[2023-09-04T06:07:17.709+0000] {standard_task_runner.py:57} INFO - Started process 7668 to run task
[2023-09-04T06:07:17.726+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-08-29T00:00:00+00:00', '--job-id', '1420', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpj_upldzm']
[2023-09-04T06:07:17.750+0000] {standard_task_runner.py:85} INFO - Job 1420: Subtask delete_data_from_table
[2023-09-04T06:07:17.960+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-08-29T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:07:18.277+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-29T00:00:00+00:00'
[2023-09-04T06:07:18.280+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-08-29' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:07:18.303+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:07:18.321+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:07:18.336+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-08-29' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:07:18.342+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:07:18.360+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220829T000000, start_date=20230904T060717, end_date=20230904T060718
[2023-09-04T06:07:18.538+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:07:18.651+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
